docs/core/versions/binding.md

```diff
@@ -0,0 +1,117 @@
+
+The following example shows the `global.json` syntax:
+
+``` json
+{
+  "sdk": {
+    "version": "2.0.0"
+  }
+}
+```
+
+The process for selecting an SDK version is:
+
+- `dotnet` searches for a `global.json` file iteratively reverse-navigating the path upward from the current working directory.
+- `dotnet` uses the SDK specified in the first `global.json` found.
+- `dotnet` binds to the latest installed SDK if no `global.json` is found.
+
+## Target Framework Monikers define build time APIs
+
+You build your project against APIs defined in a **Target Framework Moniker** (TFM). You specify the target framework in the project file. Set the `TargetFramework` element in your project file as shown in the following example:
+
+``` xml
+<TargetFramework>netcoreapp2.0</TargetFramework>
+```
+
+You may build your project against multiple TFMs. Setting multiple target frameworks is more common for libraries but can be done with applications as well. You specify a `TargetFrameworks` property (plural of `TargetFramework`). The target frameworks will be semicolon-delimited as shown in the following example:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+```
+
+A given SDK supports a fixed set of frameworks, capped to the target framework of the runtime it ships with. For example, the .NET Core 2.0 SDK includes the .NET Core 2.0 runtime, which is an implementation of the `netcoreapp2.0` target framework. The .NET Core 2.0 SDK will support `netcoreapp1.0`, `netcoreapp1.1`, and `netcoreapp2.0` but not `netcoreapp2.1` (or higher). You install the .NET Core 2.1 SDK to build for `netcoreapp2.1`
+
+.NET Standard target frameworks are also capped in the same way. The .NET Core 2.0 SDK is capped to `netstandard2.0`.
+
+You can override the minimum runtime patch version (to higher or lower versions) in the project file, as shown in the following example:
+
+``` xml
+<RuntimeFrameworkVersion>2.0.4</RuntimeFrameworkVersion>
+```
+
+ The `RuntimeFrameworkVersion` element  overrides the default version policy. When you target multiple frameworks, you must specify the minimum runtime patch version to the specific target framework, as shown in the following example for .NET Core:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+<RuntimeFrameworkVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.4</RuntimeFrameworkVersion>
+```
+
+## dotnet run rolls forward
+
+You run an application from source with [`dotnet run`](../tools/dotnet-run.md). `dotnet run` both builds and runs an application.
+
+The .NET CLI chooses the latest patch version installed on the machine. For example, if you specified `netcoreapp2.0` in your project file, and `2.0.4` is the latest .NET runtime installed, the `2.0.4` runtime is used.
```

Comment:
I wouldn't say it's the .NET CLI that chooses this.  Usually when we say CLI we mean the .NET Core SDK, but this logic is part of the host, and applies even if you have only the runtime but not the SDK installed.

Reply:
I also added a definition of **host**. I tried to future-proof it a bit, but I think it will need a small update when [this](https://github.com/dotnet/cli/issues/6237#issuecomment-398181320) arrives.